zerver/webhooks/slack/fixtures/message_slack_fixture_body_1.txt

```diff
@@ -0,0 +1 @@
+token=T54nYBAKNdWAM5ncQGDg0Pr3&team_id=T05T94349NK&team_domain=praestantiaxworkspace&service_id=6181546849778&channel_id=C065B25GE74&channel_name=zulip_web_local_host&timestamp=1703159687.522999&user_id=U05UCKN20SC&user_name=mehtabsinghcse&text=slack_bot+Testing+the+slack_webhook_new_function_1&trigger_word=slack_bot
```

Comment:
Yes, you need to add a test for that case. I'm not sure if that will involve updating that existing URL template or doing something else. You don't want to break the other test cases.

Reply:
Is there any way to get the `user_specified_topic` without introducing the `topic` query parameter to make this unit test?
Because whenever I tested my code in `Integrations Developer Panel`, the URL_TEMPLATE always goes like:
```python
/api/v1/external/slack?api_key={api_key}&stream={stream}&topic={user_specified_topic}
```

Reply:
@laurynmm 
I had another question in mind, what shall happen if the user has given a `user_specified_topic` as well as `channel_map_to_topics=0` in the URL, what shall be output in that case?
Should the `topic` parameter pick up the `user_specified_topic` in that case ?
Or should the `topic` parameter pick up `message from slack` ?